File: comms/push-notifications/apps/update-app/update-app.java
package com.example.usage;

import com.twilio.sdk.TwilioComms;
import com.twilio.sdk.resources.pushnotifications.apps.requests.AppsUpdateRequest;

public class Example {
    public static void main(String[] args) {
        TwilioComms client = TwilioComms
            .builder()
            .credentials("TWILIO_ACCOUNT_SID", "TWILIO_AUTH_TOKEN")
            .build();

        client.pushNotifications().apps().update(
            "appName",
            AppsUpdateRequest
                .builder()
                .isDefault(true)
                .build()
        );
    }
}

File: comms/push-notifications/apps/update-app/update-app.cs
using Twilio.Comms;
using System.Threading.Tasks;
using Twilio.Comms.PushNotifications;

public partial class Examples
{
    public async Task Example() {
        var client = new TwilioComms(
            accountId: "TWILIO_ACCOUNT_SID",
            authToken: "TWILIO_AUTH_TOKEN"
        );

        await client.PushNotifications.Apps.UpdateAsync(
            "appName",
            new AppsUpdateRequest {
                IsDefault = true
            }
        );
    }

}
